tests/test_core.py

```diff
@@ -574,16 +579,21 @@ def test_dagobah_expand_complex_job():
     job_3.add_edge('C**', 'D**')
 
     graph, tasks = job_1.expand(job_1.graph, job_1.tasks)
+    d = dagobah.JIJ_DELIM
     ideal_result = {
-        'A': set(['C_A*']),
-        'B': set(['C_A*']),
-        'C_A*': set(['C_C_B*_A**', 'C_C_B*_B**', 'C_C_B*_C**', 'C_C_C*']),
-        'C_C_B*_A**': set(['C_C_B*_D**']),
-        'C_C_B*_B**': set(['C_C_B*_D**']),
-        'C_C_B*_C**': set(['C_C_B*_D**']),
-        'C_C_B*_D**': set(['C_C_C*', 'C_C_D*']),
-        'C_C_C*': set(['C_C_D*']),
-        'C_C_D*': set(['D', 'E']),
+        'A': set(['C{0}A*'.format(d)]),
+        'B': set(['C{0}A*'.format(d)]),
+        'C{0}A*'.format(d): set(['C{0}B*{0}A**'.format(d),
+                                 'C{0}B*{0}B**'.format(d),
+                                 'C{0}B*{0}C**'.format(d),
+                                 'C{0}C*'.format(d)]),
+        'C{0}B*{0}A**'.format(d): set(['C{0}B*{0}D**'.format(d)]),
+        'C{0}B*{0}B**'.format(d): set(['C{0}B*{0}D**'.format(d)]),
+        'C{0}B*{0}C**'.format(d): set(['C{0}B*{0}D**'.format(d)]),
+        'C{0}B*{0}D**'.format(d): set(['C{0}C*'.format(d),
+                                       'C{0}D*'.format(d)]),
+        'C{0}C*'.format(d): set(['C{0}D*'.format(d)]),
+        'C{0}D*'.format(d): set(['D', 'E']),
```

Comment:
yeah, I did this when I wasn't solid on how the delimiter was gonna be, but true. It definitely looks more complex than it should be